solr/bin/solr

```diff
@@ -2172,6 +2172,12 @@ function start_solr() {
     SOLR_OPTS+=($AUTHC_OPTS)
   fi
 
```

Comment:
```suggestion

# If a heap dump directory is specified, enable it in SOLR_OPTS
if [[ -z "$SOLR_HEAP_DUMP_DIR" ]] && [[ "$SOLR_HEAP_DUMP" == "true" ]]; then
  SOLR_HEAP_DUMP_DIR="${SOLR_LOGS_DIR}/dumps"
fi

```

I like the idea of having a boolean flag that lets you say that you want a heap dump at the default location of heap dumps. Not a necessity, just a thought.